libcxx/test/std/language.support/support.dynamic/new.delete/new.delete.array/sized_delete_array14.pass.cpp

```diff
@@ -11,6 +11,10 @@
 // These compiler versions don't enable sized deallocation by default.
 // UNSUPPORTED: clang-17, clang-18
 
+// Android clang-r536225 identifies as clang-19 but it predates the real
+// LLVM 19.0.0, so it also leaves sized deallocation off by default.
+// UNSUPPORTED: android && clang-19
```

Comment:
Could we instead do `UNSUPPORTED: clang-19.0`? Also, is there a reason you can't update to a release version?

Reply:
AFAIK `UNSUPPORTED: clang-19.0` would work. I'll double-check.

The Android libc++ CI uses a build of LLVM from the Android LLVM team itself because the ordinary Linux LLVM builds don't have the Android builtin archives, and the Android team doesn't use the official upstream LLVM release branches. I believe the issue is that release schedules don't line up? The android_llvm README has some comments about the Android LLVM release schedule, https://android.googlesource.com/toolchain/llvm_android/+/refs/heads/main#android-clang_llvm-toolchain.

In any case, the Android LLVM build also has a large number of cherry picks and local patches. (https://android.googlesource.com/toolchain/llvm_android/+/refs/heads/main/patches/).